plugins/source/launchdarkly/resources/services/auditlog/auditlog_entries.go

```diff
@@ -0,0 +1,87 @@
+package auditlog
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/cloudquery/cloudquery/plugins/source/launchdarkly/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	ldapi "github.com/launchdarkly/api-client-go/v11"
+)
+
+func AuditLogEntries() *schema.Table {
+	return &schema.Table{
+		Name:        "launchdarkly_auditlog_entries",
+		Description: `https://apidocs.launchdarkly.com/tag/Audit-log#operation/getAuditLogEntries`,
+		Resolver:    fetchAuditLogEntries,
+		Transform:   transformers.TransformWithStruct(&ldapi.AuditLogEntryListingRep{}, transformers.WithSkipFields("Id", "Date"), transformers.WithSkipFields("Links")),
+		Columns: schema.ColumnList{
+			{
+				Name:     "id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("Id"),
```

Comment:
If we do this to avoid having `_id` we should do it for all tables

Reply:
Fair, done in [`1c138f2` (#7191)](https://github.com/cloudquery/cloudquery/pull/7191/commits/1c138f2c31159033f649e053b6b47fb0181c9fd1)